app/v2/services/RetrieveHistoricFhlUkPropertyAnnualSubmissionService.scala

```diff
@@ -37,13 +37,15 @@ class RetrieveHistoricFhlUkPropertyAnnualSubmissionService @Inject()(connector:
 
   private val downstreamErrorMap =
```

Comment:
Can we move the **downstreamErrorMap**  into a companion object and then import into both service and tests to avoid duplication of errorMappings in both code + tests : https://github.com/hmrc/property-business-api/blob/b3d5974e1602261d3e282933b6fe473fc2e2a82a/app/v2/services/RetrieveHistoricNonFhlUkPropertyPeriodSummaryService.scala#L54-L67